pkg/operator/controller/sync-http-error-code-configmap/httperrorcode_configmap.go

```diff
@@ -0,0 +1,147 @@
-sizing: border-box;\r\n    box-sizing: border-box;\r\n  }\r\n  html {\r\n    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);\r\n  }\r\n  body {\r\n    font-family: \"Helvetica Neue\", Helvetica, Arial, sans-serif;\r\n    line-height: 1.66666667;\r\n    font-size: 13px;\r\n    color: #333333;\r\n    background-color: #ffffff;\r\n    margin: 2em 1em;\r\n  }\r\n  p {\r\n    margin: 0 0 10px;\r\n    font-size: 13px;\r\n  }\r\n  .alert.alert-info {\r\n    padding: 15px;\r\n    margin-bottom: 20px;\r\n    border: 1px solid transparent;\r\n    background-color: #f5f5f5;\r\n    border-color: #8b8d8f;\r\n    color: #363636;\r\n    margin-top: 30px;\r\n  }\r\n  .alert p {\r\n    padding-left: 35px;\r\n  }\r\n  a {\r\n    color: #0088ce;\r\n  }\r\n\r\n  ul {\r\n    position: relative;\r\n    padding-left: 51px;\r\n  }\r\n  p.info {\r\n    position: relative;\r\n    font-size: 15px;\r\n    margin-bottom: 10px;\r\n  }\r\n  p.info:before, p.info:after {\r\n    content: \"\";\r\n    position: absolute;\r\n    top: 9%;\r\n    left: 0;\r\n  }\r\n  p.info:before {\r\n    content: \"i\";\r\n    left: 3px;\r\n    width: 20px;\r\n    height: 20px;\r\n    font-family: serif;\r\n    font-size: 15px;\r\n    font-weight: bold;\r\n    line-height: 21px;\r\n    text-align: center;\r\n    color: #fff;\r\n    background: #4d5258;\r\n    border-radius: 16px;\r\n  }\r\n\r\n  @media (min-width: 768px) {\r\n    body {\r\n      margin: 4em 3em;\r\n    }\r\n    h1 {\r\n      font-size: 2.15em;}\r\n  }\r\n\r\n  </style>\r\n  </head>\r\n  <body>\r\n    <div>\r\n      <h1>Application or Document Not Found</h1>\r\n      <p>No application was found at the provided URL.</p>\r\n\r\n      <div class=\"alert alert-info\">\r\n        <p class=\"info\">\r\n          Possible reasons you are seeing this page:\r\n        </p>\r\n        <ul>\r\n          <li>\r\n            <strong>Moving a page.</strong>\r\n              If you recently added or moved a page, it's possible that the page was placed in the wrong folder.\r\n          </li>\r\n          <li>\r\n            <strong>Moving a page's directory.</strong>\r\n              Sometimes the page itself may not be the cause of a 404 — it could be the page's containing folder.\r\n          </li>\r\n          <li>\r\n            <strong>The host doesn't exist.</strong>\r\n            Make sure the hostname was typed correctly and that a route matching this hostname exists.\r\n          </li>\r\n          <li>\r\n            <strong>The host exists, but doesn't have a matching path.</strong>\r\n            Check if the URL path was typed correctly and that the route was created using the desired path.\r\n          </li>\r\n          <li>\r\n            <strong>Route and path matches, but all pods are down.</strong>\r\n            Make sure that the resources exposed by this route (pods, services, deployment configs, etc) have at least one pod running.\r\n          </li>\r\n        </ul>\r\n      </div>\r\n    </div>\r\n  </body>\r\n</html>\r\n`
```

Comment:
Name these `default503ErrorPage` and `default404ErrorPage`, respectively.  

We can get this in a follow-up.